openedx/adg/lms/applications/__init__.py

```diff
@@ -1,5 +1,42 @@
 """
-init file for applications
-"""
+This app adds the ability to submit, review and manage applications for users and managers.
+
+Application hub
+------------
+ADG provides an application hub where users (learners) can go and submit application for review.
+Each user can submit only one application. Application submission process on Application hub
+consists of two mandatory steps i.e. prerequisite courses and written application.
+
+Prerequisite courses
+------------
+Application hub shows (configured) courses in its dashboard which are mandatory to pass in order to submit
+the application. Each course is available in two languages i.e Arabic and English.
```

Comment:
We can avoid this extra info that each course is available in two languages .....? as it is covered in the section below